developer-docs/latest/concepts/sso.md

```diff
@@ -0,0 +1,320 @@
+# Single Sign On
+
+Single-Sign-On on Strapi allows you to configure additional sign-in and sign-up methods for your administration panel.
+
+## Prerequisites
+
+- A Strapi application running on version 3.5.0 or higher is required.
+- To configure SSO on your application, you will need an EE license with a Gold plan.
+- Make sure Strapi is part of the applications you can access with your provider. For example, with Microsoft (Azure) Active Directory, you must first ask someone with the right permissions to add Strapi to the list of allowed applications. Please refer to your provider(s) documentation to learn more about that.
+
+## Usage
+
+SSO configuration lives in the server configuration of your application found within `/config/server.js`.
+
+### Accessing the configuration
+
+The providers' configuration should be written within the `admin.auth.providers` path of the server configuration.
+
+`admin.auth.providers` is an array of [provider configuration](#provider-configuration).
+
+```javascript
+module.exports = ({ env }) => ({
+  // ...
+  admin: {
+    // ...
+    auth: {
+      providers: [], // The providers' configuration lives there
+    },
+  },
+});
+```
+
+### Provider Configuration
+
+A provider's configuration is a Javascript object built with the following properties:
+
+| Name             | Required | Type     | Description                                                                    |
+|----------------- |----------|----------|--------------------------------------------------------------------------------|
+| `uid`            | true     | string   | The UID of the strategy. It must match the strategy's name                     |
+| `displayName`    | true     | string   | The name that will be used on the login page to reference the provider        |
+| `icon`           | false    | string   | An image URL. If specified, it will replace the displayName on the login page  |
+| `createStrategy` | true     | function | A factory that will build and return a new passport strategy for your provider. Takes the strapi instance as parameter |
+
+::: tip
+The `uid` property is the unique identifier of each strategy and is generally found in the strategy's package. If you are not sure of what it refers to, please contact the maintainer of the strategy.
+:::
+
+#### The `createStrategy` Factory
+
+A passport strategy is usually built by instantiating it using 2 parameters: the configuration object, and the verify function.
+
+##### Configuration Object
```

Comment:
This section is not clear to me. I would rephrase it.

Saying that a `callback URL` is needed in the configuration of every provider.
This information is set on the provider side when you create an auth application.
To not duplicate the information (in Strapi config and in the SSO provider auth app) we created this service function `getStrategyCallbackURL` to get the `callback URL` set in the SSO provider auth app.

The first time I tried, it was not clear the `callback URL` I have to set when I created the SSO provider auth app (in my case GitHub)
I think it's worth it to indicate the `callback URL` format to set in the SSO provider auth app.

Reply:
In fact, a callback URL is not needed for *every* provider (really often though).
I'll add a note about the structure of the URL and its usage though.